warehouse/accumulo-extensions/src/test/java/datawave/ingest/table/volumeChoosers/ShardedTableDateBasedTieredVolumeChooserTest.java

```diff
@@ -1,8 +1,7 @@
 package datawave.ingest.table.volumeChoosers;
 
-import static org.junit.Assert.*;
+import static junit.framework.TestCase.assertTrue;
```

Comment:
`junit.framework` is deprecated. Looks like intellij defaulted to the wrong import.

Reply:
Thanks, will fix.